website/static/js/osfHelpers.js

```diff
@@ -362,6 +362,15 @@ var mapByProperty = function(list, attr) {
 };
 
 
+/** 
+ * Replaces encoded & with decoded values
+ */
+var decodeText = function(text) {
+    if(typeof text === 'string') {
+        return text.replace(/&amp;/g, '&');
+    }
+    return text;
```

Comment:
Discussed in person: All existing callers of this function should be passing a string. The typecheck here is just a failsafe.